.github/workflows/ci.yml

```diff
@@ -79,6 +79,14 @@ jobs:
           - name: mxnet (Python 3.7)
             framework: mxnet
             python: 3.7
+          - name: TensorFlow+Lingvo 2.1.0v1 (Keras 2.3.1 Python 3.6)
+            framework: tensorflow2v1
+            python: 3.6
+            tensorflow: 2.1.0
+            tenorflow_gpu: 2.1.0
```

Comment:
Is the label `tensorflow_gpu` still required? I think it's covered by label `tensorflow`.

Reply:
Yes, you are right. 

I also checked the `setup.py` of the Lingvo library and they install `tensorflow-gpu` via `tensorflow-gpu==tf.__version__` (see [setup.py#L43](https://github.com/tensorflow/lingvo/blob/55263937599fdbbfb959e8eab7a40fa0aa74ad9c/pip_package/setup.py#L43)). So we can stick to the same snippets we use for the other Tensorflow workflows.